custom_components/ha_washdata/frontend.py

```
@@ -63,70 +75,229 @@ class LovelaceResourceItem(TypedDict, total=False):
+    ``served_asset_report``.
+    """
+    served = _resolve_asset(source_name, www)
+    _ensure_gzip(served)
+    try:
+        _SERVED_ASSETS[source_name] = {
+            "serving": served.name,
+            "minified": served.name != source_name,
+            "bytes": served.stat().st_size,
+        }
+    except OSError:
+        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)